Source/WebCore/PAL/pal/spi/cocoa/PassKitSPI.h

```diff
@@ -335,12 +335,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 typedef void(^PKCanMakePaymentsCompletion)(BOOL isValid, NSError *);
 
-#if HAVE(PASSKIT_DEFERRED_PAYMENTS)
-@interface PKDeferredPaymentRequest (Staging_104652810)
-@property (nonatomic, strong, nullable) NSTimeZone *freeCancellationDateTimeZone;
-@end
```

Comment:
Doesn't this need to be kept around in the `!USE(APPLE_INTERNAL_SDK)` section for open source builds?

Reply:
Never mind, this is in the public SDK.